Automattic/jetpack
# webpack-config: Replace duplicate-package-checker-webpack-plugin

## Proposed changes:

The package has not been updated for 5 years, and causes Webpack 5 to raise warnings about "Module.issuer: Use new ModuleGraph API".

A maintained replacement seems to be
@cerner/duplicate-package-checker-webpack-plugin, so let's go with that.

### Other information:


## Jetpack product discussion


Fixes #20792.

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Does the build no longer have "Module.issuer: Use new ModuleGraph API" warnings?
* Are duplicate packages in the build still detected?
